tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "emitDecoratorMetadata": true,
     "extendedDiagnostics":true,
     "noErrorTruncation": true,
-    "sourceMap": true
+    "sourceMap": true,
```

Comment:
you might want to fix your editor's linter, because trailing commas are not supported by json format afaik

Reply:
uhh nothing seems to be broken